test/integration/targets/connection_psrp/tests.yml

```diff
@@ -59,3 +59,42 @@
     assert:
       that:
       - async_out.stdout_lines == ["abc"]
+
+- name: Output unicode characters from Powershell using PSRP
+  raw: |
+    Write-Host '💩'
+  register: raw_unicode_output
+
+- name: Assert unicode output
+  assert:
+    that:
+    - raw_unicode_output is changed
+    - raw_unicode_output.rc == 0
+    - raw_unicode_output.stdout == '💩'
+    - raw_unicode_output.stderr == ''
+
+- name: Output unicode characters from Powershell using PSRP
+  win_command: |
+    powershell.exe -ExecutionPolicy ByPass -Command "Write-Host '💩'"
+  register: command_unicode_output
+
+- name: Assert unicode output
+  assert:
+    that:
+    - command_unicode_output is changed
+    - command_unicode_output.rc == 0
+    - command_unicode_output.stdout == '💩'
+    - command_unicode_output.stderr == ''
+
+- name: Output unicode characters from Powershell using PSRP
+  win_shell: |
+    Write-Host '💩'
+  register: shell_unicode_output
+
+- name: Assert unicode output
+  assert:
+    that:
+    - shell_unicode_output is changed
+    - shell_unicode_output.rc == 0
+    - shell_unicode_output.stdout == '💩'
```

Comment:
`"shell_unicode_output.stdout == '\U0001F4A9\n'". Need to use the `\U` form and add `\n` to the end.